pages/LanguageSelectionPage.py

```diff
@@ -0,0 +1,24 @@
+from framework.common import Page, PageObject
+from framework.selenium_plus import *
+
+
+@PageObject
+class _LanguageSelectionPage(Page):
+    class ID:
+        #TO-DO  add ID
+        btnsubmit = "button#continue-action"
+
+    def language(self):
```

Comment:
Each language will have its own method for now and one common method where we can send the language type also as an argument